homework6/src/main/java/Main.java

```diff
@@ -0,0 +1,300 @@
+     * @return boolean value
+     */
+    private static boolean checkIf(List<Car> carList) {
+        return carList
+                .stream()
+                .map(s -> s.getPower())
+                .anyMatch(s -> s > 250);
+    }
+
+    /**
+     * this method sorts the List in reverse order
+     *
+     * @param set Set of String
+     * @return List of String
+     */
+    private static List<String> sortingStringsInReverseOrder(Set<String> set) {
+        return set
+                .stream()
+                .sorted(Comparator.comparing(String::valueOf).reversed())
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * this method sorts the List by the "make"
+     *
+     * @param set Set of Car
+     * @return List of Car
+     */
+    private static List<Car> sortingCustomObjects(Set<Car> set) {
+        return set
+                .stream()
+                .sorted(Comparator.comparing(Car::getMake).reversed())
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * this method creates a Map with random numbers, and define the number of elements, which have value over 10
+     *
+     * @param n the number of the Integers
+     */
+    private static void nElementInMap(int n) {
+        Random random = new Random();
+        int index = 1;
+        Map<Integer, Integer> map = new HashMap<>();
+
+        while (index <= n) {
+            map.put(index, random.nextInt());
+            index++;
+        }
+
+        System.out.println("\nThe content of the map: " + map);
+
+        long nr = map
+                .entrySet()
+                .stream()
+                .filter(i -> i.getValue() > 10)
+                .count();
+
+        System.out.println("Number of elements have value over 10: " + nr);
+
+    }
+
+    /**
+     * this method counts the elements of the list, calculates the exponential to the numbers,
+     * converts the List to Map, and then prints the elements out
+     *
+     * @param randomNumbersInList List of random Integer
+     */
+    private static void randomNumbersInMap(List<Integer> randomNumbersInList) {
+        List<Double> list = randomNumbersInList
+                .stream()
+                .mapToDouble(s -> Math.log1p((double) s))
+                .boxed()
+                .collect(Collectors.toList());
+
+        long number = randomNumbersInList
+                .stream()
+                .count();
+
+        int i = 0;
+        Map<Integer, Double> result = new HashMap<>();
+        for (Double element : list) {
+            result.put(++i, element);
+        }
+
+        System.out.println("\nRandom numbers: ");
+        randomNumbersInList.forEach(System.out::println);
+        System.out.println("\nNumber of random numbers of Integer: " + number);
+        System.out.println("\nThe exponential to the numbers: ");
+        result.entrySet().forEach(entry -> {
+            System.out.println(entry.getKey() + " " + entry.getValue());
+        });
```

Comment:
a bit complicated from what was requested, keep it as simple as possible (you only had to map the current value to an exp / pow function...and count)